core-galleon-pack/wildfly-feature-pack-build.xml

```diff
@@ -0,0 +1,126 @@
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<build xmlns="urn:wildfly:feature-pack-build:2.0">
+    <default-packages>
+        <package name="modules.all"/>
+        <package name="docs"/>
+    </default-packages>
+    <package-schemas>
+        <group name="org.wildfly.core"/>
+    </package-schemas>
+
+    <config model="standalone">
+        <props>
+            <prop name="config.branch-is-batch" value="true"/>
+            <prop name="--admin-only" value=""/>
+            <prop name="--internal-empty-config" value=""/>
+            <prop name="--internal-remove-config" value=""/>
+        </props>
+        <packages>
+            <package name="product.conf" optional="true"/>
+            <package name="misc.standalone"/>
+            <package name="org.jboss.as.standalone.main"/>
+
+            <!--
+             This one is here to avoid
+             ERROR [org.jboss.as.domain.http.api.undertow] (MSC service thread 1-4) WFLYDMHTTP0004: Unable to load error context for slot main, disabling error context.
+              -->
+            <package name="org.jboss.as.domain-http-error-context.main"/>
+
+            <!-- cleanup runtime dirs -->
+            <package name="cleanup.standalone.config.history.dir" optional="true"/>
+            <package name="cleanup.standalone.log.dir" optional="true"/>
+            <package name="cleanup.standalone.data.dir" optional="true"/>
+            <package name="cleanup.standalone.tmp.vfs" optional="true"/>
+        </packages>
+    </config>
+
+    <config model="domain">
+        <props>
+            <prop name="config.branch-is-batch" value="true"/>
+            <prop name="--empty-domain-config" value=""/>
+            <prop name="--remove-existing-domain-config" value=""/>
+            <prop name="--empty-host-config" value=""/>
+            <prop name="--remove-existing-host-config" value=""/>
+        </props>
+        <packages>
+            <package name="product.conf" optional="true"/>
+            <package name="misc.domain"/>
+            <package name="org.jboss.as.domain-management.main"/>
```

Comment:
This package must be present in any server. Despite its name It is not specific to the 'domain' config model. Just like the JIRA component formally known as 'Domain Management' was not specific to domain mode. :)

Reply:
Oh, sure, added to standalone as well, thanks!